profitbricks-rest/src/test/java/org/apache/jclouds/profitbricks/rest/binder/volume/CreateVolumeRequestBinderTest.java

```diff
@@ -42,6 +42,7 @@ public void testCreatePayload() {
               .dataCenterId("datacenter-id")
               .name("jclouds-volume")
               .size(3)
+              .type("HDD")
```

Comment:
Should an enum work better here, so users have a constrained set of options to use?